src/utils/daos.c

```diff
@@ -1359,10 +1359,8 @@ help_hdlr(int argc, char *argv[], struct cmd_args_s *ap)
 	if (argc <= 2) {
 		FIRST_LEVEL_HELP();
 	} else if (strcmp(argv[2], "filesystem") == 0 || strcmp(argv[2], "fs") == 0) {
-		if (argc == 3) {
+		if (argc == 3 || strcmp(argv[3], "copy") == 0) {
 			ALL_FS_CMDS_HELP();
```

Comment:
@mchaarawi @dsikich Actually, I don't think we can do this, because then we don't get to see the --src and --dst options for copy.
Maybe?
```suggestion
		if (argc == 3 || strcmp(argv[3], "copy") == 0) {
			FS_COPY_CMDS_HELP();
```

Reply:
@daltonbohning updated.